README.md

```diff
@@ -1 +1,30 @@
-# hello-microverse-project
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# github-flow
+
+> This is an example of github flow. Add new feature branch and make commits.
+
+
+## Built With
+
+- HTML, CSS, GIT
+
+
+## Authors
+
+👤 **Alejandro Maggioni**
+
+- GitHub: [@githubhandle](https://github.com/alex1779/)
+- Twitter: [@twitterhandle](https://twitter.com/alex1779)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/alejandro-maggioni-086678b5/)
+
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
Please Use this [Updated Readme template ](https://github.com/microverseinc/readme-template). 
* The only **optional** section is FAQ.
* you should **remove the section description**, i.e. "> Describe the tech stack and include only the relevant sections that apply to your project." should be removed and filled in with your custom content.
* you should remove **template logo**. Optionally, you can replace with your own logo if you have one.